src/buyingcatalogue/values.yaml

```diff
@@ -24,6 +24,51 @@ azurite:
     queuePort: 10001
     tablePort: 10002
 
+db:
+  enabled: false
+  disabledUrl: 
+  replicaCount: 1
+  useLocalImage: false
+  nameOverride: "db"
+  fullnameOverride: "gpitfutures-bc-db"
+  service:
+    type: ClusterIP
+    port: 1433
```

Comment:
I think it might be best for this to be a non standard port, just so when running locally it doesn't clash if someone has sql installed.